teams/leadership-council.toml

```diff
@@ -0,0 +1,36 @@
+name = "leadership-council"
+
+[people]
+leads = []
+members = [
+    "carols10cents",
+    "eholk",
+    "ehuss",
+    "jackh726",
+    "jonathanpallant",
+    "khionu",
+    "Mark-Simulacrum",
+    "m-ou-se",
+    "rylev",
+]
+alumni = []
+
+[[github]]
+orgs = ["rust-lang", "rust-lang-nursery"]
+
+[rfcbot]
+label = "T-leadership-council"
+name = "Leadership Council"
+ping = "rust-lang/leadership-council"
+
+[website]
+name = "Leadership council"
+description = "Charged with the success of the Rust Project as whole, consisting of representatives from top-level teams"
+email = "council@rust-lang.org"
+weight = 1000
+
+[[lists]]
+address = "council@rust-lang.org"
+
+[[zulip-groups]]
+name = "council"
```

Comment:
Nit: should we use `T-council`? The council isn't necessarily a team in the traditional sense though...

Reply:
The *stream* is just `council`, so having the *group* just be `council` keeps it consistent.